lua/telescope/pickers.lua

```diff
@@ -491,6 +491,44 @@ function Picker:hide_preview()
   -- 2. Resize prompt & results windows accordingly
 end
 
+-- A simple interface to remove an entry from the results window without closing
```

Comment:
Emmy documentation uses 3 slashes 

https://github.com/tjdevries/tree-sitter-lua/blob/master/HOWTO.md#parameter

Maybe at some point we can export the documentation for `pickers.lua`